src/buzz/buzzparser.c

```diff
@@ -674,6 +680,8 @@ int parse_while(buzzparser_t par) {
    fetchtok();
    /* If the condition is false, jump to the end */
    chunk_append("\tjumpz " LABELREF "%u", wend);
+   // Allow the use of non-cuddled brackets
```

Comment:
Please avoid the C++ comment style. Use `/* comment */` instead of `// comment`.